analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -142,35 +142,39 @@ class Analyzer(private val config: AnalyzerConfiguration) {
 
                             // By convention, project ids must be of the type of the respective package manager.
                             results.forEach { (_, result) ->
-                                require(result.project.id.type == manager.managerName) {
-                                    "Project '${result.project.id.toCoordinates()}' must be of type " +
-                                            "'${manager.managerName}'."
+                                val invalidProjects = result.filter { it.project.id.type != manager.managerName }
+                                require(invalidProjects.isEmpty()) {
+                                    val projectString =
+                                        invalidProjects.joinToString { "'{${it.project.id.toCoordinates()}}'" }
```

Comment:
Are the outer curly braces intended here?

Reply:
No, fixed.